Hyper/Camera.cs

```diff
@@ -147,9 +158,33 @@ private void UpdateVectors()
             _up = Vector3.Normalize(Vector3.Cross(_right, _front));
         }
 
-        public void Move(KeyboardState input, float time)
+        private Matrix4 TranslateMatrix(Vector4 to)
         {
-            const float cameraSpeed = 1.5f;
+            Matrix4 T;
+            if (Curve != 0)
```

Comment:
why have != 0 here but abs(Curve) < 0.0001 in the other method. I don't think it makes a difference and we should be consistant.